packages/drizzle/src/queries/getTableColumnFromPath.ts

```diff
@@ -241,36 +268,13 @@ export const getTableColumnFromPath = ({
             // this is fine, not every block will have the field
           }
           if (!result) {
+            joins.pop()
```

Comment:
It seems strange to have to `pop` the join when there is !result. Is it possible to have additional `joins` added in from the `getTableColumnFromPath` 251 and in that case you'd be popping the wrong join?

If that is a possibility I'd add `let join` above the if statements, on L223 and assign it instead of push it. Then we can push it after this if block.

Does this make sense?